scripts/render_tags.py

```diff
@@ -0,0 +1,122 @@
+"""Custom rendering code for the /api/{keras_nlp|keras_cv}/models page.
+
+The model metadata is pulled from the library, each preset has a
+metadata dictionary as follows:
+
+{
+    'description': Description of the model,
+    'params': Parameter count of the model,
+    'official_name': Name of the model,
+    'path': Relative path of the model on keras.io,
+}
+"""
+
+
+TABLE_HEADER = (
+    "Preset ID | Model | Parameters | Description\n"
+    "----------|-------|------------|------------\n"
+)
+
+
+def format_param_count(metadata):
+    """Format a parameter count for the table."""
+    try:
+        count = metadata['params']
+    except KeyError:
+        return "Unknown"
+    if count >= 1e9:
+        return f"{int(count / 1e9)}B"
```

Comment:
should these be floats formatted to two decimal points

Reply:
Done.